src/application/App.tsx

```diff
@@ -81,6 +81,20 @@ function LeftSidePanelWrapper() {
     setPrimaryPane(selectedPane);
   };
 
+  // Configure keyboard shortcuts to open/close side panel
+  useEventListener('keydown', (e) => {
+    if (e.metaKey) {
+      switch (e.key.toLowerCase()) {
+        case 'b':
+          return handleSideBarClick(primaryPane);
```

Comment:
This is conflicts with cmd+B in the editor to make text bold: pressing cmd+B makes the text bold and toggles the sidebar. We need to prevent the event from bubbling out of the editor

Reply:
Nice catch. Maybe we just need to remove this shortcut.